kopi/assetlist.json

```diff
@@ -0,0 +1,525 @@
+      },
+      "images": [
+        {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/Osmosis/images/osmo.svg"
+        }
+      ],
+      "keywords": [
+        "dex",
+        "mm"
+      ],
+      "type_asset": "sdk.coin",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "osmosis",
+            "base_denom": "uosmo",
+            "channel_id": "channel-88475"
+          },
+          "chain": {
+            "channel_id": "channel-6",
+            "path": "transfer/channel-88475/uosmo"
+          }
+        }
+      ]
+    },
+    {
+      "description": "Atom from cosmoshub Chain",
+      "denom_units": [
+        {
+          "denom": "ibc/25418646C017D377ADF3202FF1E43590D0DAE3346E594E8D78176A139A928F88",
+          "exponent": 0
+        },
+        {
+          "denom": "Atom",
+          "exponent": 6
+        }
+      ],
+      "base": "ibc/25418646C017D377ADF3202FF1E43590D0DAE3346E594E8D78176A139A928F88",
+      "name": "ATOM",
+      "display": "Atom",
+      "symbol": "ATOM",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/cosmoshub/images/atom.svg"
+      },
+      "images": [
+        {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/cosmoshub/images/atom.svg"
+        }
+      ],
+      "keywords": [
+        "dex",
+        "mm"
+      ],
+      "type_asset": "sdk.coin",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "cosmoshub",
+            "base_denom": "uatom",
+            "channel_id": "channel-1253"
+          },
+          "chain": {
+            "channel_id": "channel-7",
+            "path": "transfer/channel-1253/uatom"
+          }
+        }
+      ]
+    },
+    {
+      "description": "arbitrage receipt token of cUSDT",
+      "denom_units": [
+        {
+          "denom": "uasusdtinj",
+          "exponent": 0
+        },
+        {
+          "denom": "asusdtinj",
+          "exponent": 6
+        }
+      ],
+      "base": "uasusdtinj",
+      "name": "asUSDT.inj",
+      "display": "asUSDT.inj",
+      "symbol": "asUSDT.inj",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/non-cosmos/ethereum/images/usdt.svg"
+      },
+      "images": [
+        {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/non-cosmos/ethereum/images/usdt.svg"
+        }
+      ],
+      "keywords": [
+        "dex",
+        "mm"
+      ],
+      "type_asset": "sdk.coin",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "noble",
+            "base_denom": "uusdc",
+            "channel_id": "channel-109"
+          },
+          "chain": {
+            "channel_id": "channel-0",
+            "path": "transfer/channel-109/uusdc"
```

Comment:
These traces for asUSDT.inj and asUSDC are not accurate for these assets (they describe USDC from Noble)